### PR TITLE
[fix] Fixed SSD1306::initialize() cast to bool for return 

### PR DESCRIPTION
**Pull request by TomSaw**
Fix for this warning:

`modm/src/modm/driver/display/ssd1306_impl.hpp:52:32: warning: narrowing conversion of '((modm::Ssd1306<modm::platform::I2cMaster1>*)this)->modm::Ssd1306<modm::platform::I2cMaster1>::commandBuffer[11]' from 'uint8_t' {aka 'unsigned char'} to 'bool' [-Wnarrowing]
   52 |  RF_END_RETURN(commandBuffer[11]);
modm/src/modm/processing/resumable/macros.hpp:188:29: note: in definition of macro 'RF_RETURN_0'
  188 |    return {modm::rf::Stop, (result)}; \`

### CODE REVIEW

**Review by rleh — APPROVED** [review]

**Review by chris-durand — APPROVED** [review]

**Comment by salkinium** [comment]
Hm… the compile HAL workflow runs *every* time someone approves. So now it's running twice. I'm mildly unconviced of the usefulness of this construct… ;-P

**Comment by TomSaw** [comment]
2 times is better than half time

**Comment by rleh** [comment]
> Hm… the compile HAL workflow runs every time someone approves. So now it's running twice. I'm mildly unconviced of the usefulness of this construct… ;-P

We should switch to labels instead of the approval. I will open a PR soon...

But wait, chris-durands (he has no merge permission to this repo?!) approval also triggers the CI runs, this was somehow unexpected to me 😮
This github actions contexts are really badly documented.

**Comment by salkinium** [comment]
I'm experimenting with using workflow_dispatch on my fork, with a PR input, perhaps that has better semantics.

**Comment by salkinium** [comment]
Eh, it does not show the manually triggered workflow in the PR job lists, so we cannot use that approach. (See https://github.com/salkinium/modm/pull/1) Guess labels is the way to go?

**Pull request merged**
